sdcm/argus_results.py

```diff
@@ -276,3 +281,77 @@ def send_manager_snapshot_details_to_argus(argus_client: ArgusClient, snapshot_d
     for key, value in snapshot_details.items():
         result_table.add_result(column=key, row="#1", value=value, status=Status.UNSET)
     submit_results_to_argus(argus_client, result_table)
+
+
+def send_iotune_results_to_argus(argus_client: ArgusClient, preset_io_props, io_properties, node, params):
+    def _bottom_limit(metric_val, threshold=0.15):
+        """
+            Determine disk metric deviation threshold and
+            use that as fixed limit
+        """
+        return metric_val * threshold
+    if not argus_client:
+        LOGGER.warning("Will not submit to argus - no client initialized")
+        return
+    preset_disk = next(iter(preset_io_props.get("disks", [])), None)
+    if not preset_disk:
+        LOGGER.warning("Unable to continue - node should have io_properties.yaml, but it doesn't.")
+        TestFrameworkEvent(source="send_iotune_results_to_argus",
+                           message="Unable to continue - node should have io_properties.yaml, but it doesn't.",
+                           severity=Severity.ERROR).publish()
+        return
+
+    class IOPropertiesResultsTable(GenericResultTable):
+        class Meta:
+            name = f"{params.get('cluster_backend')} - {node.db_node_instance_type} Disk Performance"
+            description = "io_properties.yaml comparison with live data"
```

Comment:
it's no longer comparison rather pure data from io_properties (row name should define 'image preset' or 'measured')

Reply:
Fixed